Readmes for perseus & perseus-editor (#236)
Khan/perseus: packages/perseus-editor/README.md

# @khanacademy/perseus-editor

> The perseus editor package contains things that are needed for the editor, but not for rendering perseus content on its own. It is nevertheless very tightly coupled to perseus core.

Reviewer: A few nits. :) 
```suggestion
The Perseus editor package contains components for editing Perseus content, but not for rendering Perseus content on its own. It is nevertheless very tightly coupled to perseus core.
```
